dataland-frontend/src/components/resources/frameworkDataSearch/ShowReportsBanner.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="col-12 text-left bg-white mb-4 mp-0 all-sides-blue-border" data-test="reportsBanner" style="width: 99.5%">
+    <span class="font-bold text-gray-800 mt-0 mb-0 mp-0 ml-0 mr-0" style="font-size: 11pt">
+      Data extracted from the company report. Company reports:
+    </span>
```

Comment:
Das hat wieder nichts mit der Story zu tun aber in der Architekturübersicht fehlt der Dokumentenservice noch komplett. Könntet ihr den bitte noch einbauen.

Reply:
Haben das Architekturdiagramm aktualisiert.